proxy/src/telemetry/metrics/mod.rs

```diff
@@ -265,11 +262,11 @@ impl Metrics {
                 responses.push(ResponseScope {
                     ctx: status_code.map(|code| {
                         ResponseCtx {
-                            http_status_code: u32::from(code.as_u16()),
+                        http_status_code: u32::from(code.as_u16()),
                         }
                     }),
                     ends: ends,
-                    response_latencies: res_stats.latencies.into(),
+                    response_latencies: (&res_stats.latencies).into(),
```

Comment:
???

Reply:
at one point there was a reason for why this became a pointer but i definitely don't remember what it was --- i'll take a look.

Reply:
ah it's because the `into()` implementation for `Latencies` takes a pointer. at one point i think i was hoping we could re-use the array allocations or something?